src/main/webapp/community.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Community Page</title>
+    <link rel="stylesheet" href="/css/main.css">
+
+    <script>
+
+      /** Fetches users and adds them to the page. */
+      function fetchUserList(){
+        const url = '/user-list';
+        fetch(url).then((response) => {
+          return response.json();
+        }).then((users) => {
+          const list = document.getElementById('list');
```

Comment:
`list` is likely going to conflict with the name of some other list in the future. Can we update this id to be a bit more specific? Something like `user-list` or `list-of-usernames`.